src/Symfony/Installer/NewCommand.php

```diff
@@ -133,7 +139,22 @@ private function checkSymfonyVersionIsInstallable()
     {
         // 'latest' is a special version name that refers to the latest stable version
         // available at the moment of installing Symfony
-        if ('latest' === $this->version) {
+        if (self::VERSION_LATEST === $this->version) {
+            return $this;
+        }
+
+        // Get the Symfony versions list (http://symfony.com/versions.json)
+        $versionsList = $this->getVersionsList();
```

Comment:
Well, this is still only necessary if the user doesn't pass a version number in the `2.M.N` format (to either fetch the latest patch version, the LTS version or the latest stable version).